src/emailpro/account/add/emailpro-account-add.controller.js

```diff
@@ -191,6 +199,12 @@ angular.module('Module.emailpro.controllers')
     $scope.loadOrderList = function () {
       EmailPro.getOrderList($stateParams.productId).then((data) => {
         $scope.ordersList = data;
+        _.forEach($scope.ordersList, (orderAvailable) => {
```

Comment:
Maybe you could have a system in case other countries need this feature as well, because if we are asked to have this for other places, then someone will have to rewrite everything

Reply:
This logic has been moved to a function in a service. It can be modified when required.